WIRELESS_OPERATION.md

```diff
@@ -30,6 +34,10 @@ This section allows full control of the master (overall) volume and to mute/unmu
 
 Controls will be made available on a per-action or per-state basis. Shown here, the pack and wand are both in an Idle state while in the "Original" mode which allows the pack to be turned on/off remotely. The options to remotely vent or to "Attenuate" are only enabled when the devices are in a specific state.
 
+**Vent:** This can only be triggered when in the Super Hero mode and while the pack is Powered.
```

Comment:
Is "Powered" something specific in the GPStar code? Does this mean "On" as in "MODE_ON"? Perhaps better to clarify, for example, "and while the cyclotron is spinning?" or "and while the cyclotron is active?" or some other more descriptive term?

Reply:
Ah gotcha, so it's referring to something in the Attenuator's WiFi GUI? Thank you for the clarification, that makes a lot more sense.